qemu/tests/cfg/usb.cfg

```diff
@@ -262,7 +262,7 @@
                 - usb_multi_times:
                     usb_repeat_times = 5000
         - usb_multi_disk:
-            only ehci
+            only usb-ehci
```

Comment:
The usb-storage is not a low-speed  device in usb_multi_disk case,so it can be attached to usb-ehci.